src/hoi4_world/countries/hoi4_country_converter.cpp

```diff
@@ -174,6 +174,40 @@ std::vector<hoi4::EquipmentVariant> DetermineActiveVariants(const std::vector<ho
    return active_variants;
 }
 
+
+std::set<std::string> ConvertLaws(const std::set<std::string>& vic3_laws, std::string_view ideology)
+{
+   std::set<std::string> hoi4_laws;
+
+   // civilian economy with export focus is the default
+   hoi4_laws.emplace("civilian_economy");
+   hoi4_laws.emplace("export_focus");
+
+   // but fascist countries change it up
+   if (ideology == "fascism")
+   {
+      hoi4_laws.clear();
+      hoi4_laws.emplace("partial_economic_mobilisation");
+      hoi4_laws.emplace("limited_exports");
+   }
+
+   if (vic3_laws.contains("law_peasant_levies"))
+   {
+      hoi4_laws.emplace("disarmed_nation");
```

Comment:
only US starts as disarmed in hoi4. All other nations are at least "volunteer only". Balance wise, it probably maps best to volunteer only, as it's a 2% conscription rate (professional army is 1%, national militia is 5%, mass conscription is 4%)
But flavor wise, it's probably better to map it to "limited conscription".